### ISSUE TITLE
Override system theme UA styles in mobile Safari

### ISSUE THREAD

**Issue by vsandvold** [fabric-ds/issues#59]
In mobile Safari, an input search field will get a gray background color (see screenshot).

The color comes from a user agent style in newer versions of Safari on iOS. The browser chooses this color by default, because Fabric does not set a specific background color on the input element.

Here's the UA style:
```css
background-color: -apple-system-opaque-tertiary-fill;
```

This seems to be part of some OS theming support by Apple https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/color/#system-colors

I found it through here https://blog.jim-nielsen.com/2021/things-i-learned-reading-webkits-ua-stylesheet/#css-keywords

Specifying the background color for input fields in Fabric would solve this problem.

![IMG_672AD5BFD65D-1](https://user-images.githubusercontent.com/6734787/152182704-842df6c0-3a66-49e5-8a7f-e628982a983a.jpeg)

**Comment by Skadefryd** [comment]
Looks like this works as intended. The markup surrounding the mentioned input type="search" on the frontpage is not following the markup suggested in fabric.    https://css.fabric-ds.io/pages/form-elements.html       <--  The .input class on the wrapper looks like a solve for this, I dont see any bi-effects either by doing it this way.

**Comment by vsandvold** [comment]
Thanks for looking into this! Good to know that it's not an issue with Fabric.

Just for the record, the gray background color will only appear for `input[type="search"]` in Safari on iOS. The UA styles look like this:
![Skjermbilde 2022-03-18 kl  13 24 10](https://user-images.githubusercontent.com/6734787/159004224-00e45a92-756a-4f96-87a1-78e0ef266f7c.png)


Another style from Fabric that seems to affect the search input appearance is:
```
[type="search"] {
    -webkit-appearance: textfield;
}
```

Overriding this in custom styles leads to other interesting side-effects, with different border radiuses etc.

Bottom line, let's stick with the defaults in Fabric 😄
